tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_advertiser.c

```diff
@@ -262,6 +402,115 @@ static void test_tx_cb_multi(void)
 	PASS();
 }
 
+static void test_tx_send_order(void)
+{
+	struct net_buf *buf[CONFIG_BT_MESH_ADV_BUF_COUNT];
+	uint8_t xmit = BT_MESH_TRANSMIT(2, 20);
+
+	bt_init();
+	adv_init();
+
+	/* Verify sending order */
+	allocate_all_net_buf(buf, ARRAY_SIZE(buf), xmit);
+	verify_net_buf_overflow();
```

Comment:
What is the need to call `verify_net_buf_overflow` so many times? One should be enough. Otherwise looks like we don't trust our test code.

Reply:
Nor addressed.